### PR TITLE
Update CurrencyConverter.php

### PR DESCRIPTION
**Pull request by dudztroyer**
Updating from fixer to currencyconverterapi.

### CODE REVIEW

File: libraries/CurrencyConverter.php [hunk 1]
```diff
@@ -132,8 +132,8 @@ private function needToUpdateDatabase($diff, $row)
 
     private function getRates()
     {
-        $url = 'http://api.fixer.io/latest?base=' . $this->fromCurrency . '&symbols=' . $this->toCurrency;
-
+        //$url = 'http://api.fixer.io/latest?base=' . $this->fromCurrency . '&symbols=' . $this->toCurrency;
```
Comment (AlessandroMinoccheri) [inline libraries/CurrencyConverter.php]: remove comments please

File: libraries/CurrencyConverter.php [hunk 2]
```diff
@@ -144,8 +144,11 @@ private function getRates()
         if (isset($result)) {
             $conversion = json_decode($result, true);
 
-            if (isset($conversion['rates'][$this->toCurrency])) {
+           /* if (isset($conversion['rates'][$this->toCurrency])) {
```
Comment (AlessandroMinoccheri) [inline libraries/CurrencyConverter.php]: remove comments please

File: libraries/CurrencyConverter.php [hunk 3]
```diff
@@ -144,8 +144,11 @@ private function getRates()
...
                 return $conversion['rates'][$this->toCurrency];
+            }*/
+			if (isset($conversion[$this->fromCurrency . '_' . $this->toCurrency])) {
```
Comment (AlessandroMinoccheri) [inline libraries/CurrencyConverter.php]: Indent well the code please

**Comment by dudztroyer** [comment]
Done.

**Comment by AlessandroMinoccheri** [comment]
LGTM 👍 Thanks for your help

**Pull request merged**